src/status_im/contexts/wallet/send/routes/view.cljs

```diff
@@ -1,39 +1,137 @@
+       [quo/bottom-actions
+        {:button-one-label (i18n/label :t/apply-changes)
+         :button-one-props {:disabled?           (= selected-networks @network-preferences)
+                            :on-press            (fn []
+                                                   (rf/dispatch [:wallet/update-receiver-networks
+                                                                 @network-preferences])
+                                                   (rf/dispatch [:hide-bottom-sheet])
+                                                   (fetch-routes))
+                            :customization-color color}}]])))
+
 (defn route-item
-  [{:keys [amount from-network to-network status]}]
-  [rn/view {:style style/routes-inner-container}
-   [quo/network-bridge
-    {:amount  amount
-     :network from-network
-     :status  status}]
-   (if (= status :default)
-     [quo/network-link
-      {:shape           :linear
-       :source          from-network
-       :destination     to-network
-       :container-style style/network-link}]
-     [rn/view {:style {:width 73}}])
-   [quo/network-bridge
-    {:amount          amount
-     :network         to-network
-     :status          status
-     :container-style {:right 12}}]])
+  [{:keys [amount from-network to-network status theme fetch-routes]}]
+  (if (= status :add)
+    [quo/network-bridge
+     {:status          :add
+      :container-style style/add-network
+      :on-press        #(rf/dispatch [:show-bottom-sheet
+                                      {:content (fn [] [networks-drawer
+                                                        {:theme        theme
+                                                         :fetch-routes fetch-routes}])}])}]
+    [rn/view {:style style/routes-inner-container}
+     [quo/network-bridge
+      {:amount  amount
+       :network from-network
+       :status  status}]
+     (if (= status :default)
+       [quo/network-link
+        {:shape           :linear
+         :source          from-network
+         :destination     to-network
+         :container-style style/network-link}]
+       [rn/view {:style {:width 73}}])
+     [quo/network-bridge
+      {:amount          amount
+       :network         to-network
+       :status          status
+       :container-style {:right 12}}]]))
 
-(defn view
-  [{:keys [amount routes loading-networks]}]
-  (let [loading-suggested-routes? (rf/sub [:wallet/wallet-send-loading-suggested-routes?])
-        candidates                (:candidates routes)]
-    (if (or (and (not-empty loading-networks) loading-suggested-routes?) (not-empty candidates))
+(defn- find-affordable-networks
+  [{:keys [balances-per-chain]} input-value selected-networks]
+  (->> balances-per-chain
+       (filter (fn [[_ {:keys [balance chain-id]}]]
+                 (and
+                  (>= (js/parseFloat balance) input-value)
+                  (some #(= % chain-id) selected-networks))))
+       (map first)))
```

Comment:
Maybe this should be in utils file?

Reply:
This function is local to this file. The utils file in under a `common` namespace, so I think it is better to keep helper functions that are not used across multiple namespaces to their local files